### ISSUE TITLE
Remove \" in triple quote (decode json fails)

### ISSUE THREAD

**Issue by laingit** [avh4/elm-format#384]
> Before

json : String
json =
    """
    {
      "text": "String inside \\"why?\\"  "
    }
    """

> After

json : String
json =
    """
    {
      "text": "String inside "why?"  "
    }
    """

**Comment by avh4** [comment]
This is correct behavior; both of your examples are parsed as the same value by `elm-make`/`elm-compiler`, and your before example will also fail to parse as valid JSON.

elm-format is simply making the behavior of `elm-make` more clear.  You will need to escape the backslashes to get a valid JSON string:

```
"""{"text": "String inside \\"why?\\" "}"""
```

**Comment by avh4** [comment]
(Also created https://github.com/avh4/elm-format/issues/387 for discussion of whether elm-format should diverge from elm-compiler regarding this.)
